cmd/estuary-shuttle/main.go

```diff
@@ -1210,6 +1210,10 @@ func (s *Shuttle) ServeAPI() error {
 	admin.GET("/net/rcmgr/stats", s.handleRcmgrStats)
 	admin.GET("/system/config", s.handleGetSystemConfig)
 
+	storageProvider := e.Group("/storage-provider")
+	storageProvider.Use(s.AuthRequired(util.PermLevelAdmin))
+	admin.GET("/list/:n", s.handleStorageProviderList)
```

Comment:
I think this should be `storageProvider.GET`, not `admin.GET`

Reply:
good catch. No idea why I wrote admin here 😅